index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const arithmetic = require('./lib/arithmetic.js');
+const greet = require('./lib/greet.js');
+
+
```

Comment:
what I was saying is invoke he exports function here. for best practice.
like
arithmetic.add(1,10);
arithmetic.sub(10,1);
greet('Zachary');